src/translations/components/TranslationFields/TranslationFields.tsx

```diff
@@ -166,6 +168,11 @@ const TranslationFields = (props: TranslationFieldsProps) => {
                         saveButtonState="default"
                         onDiscard={onDiscard}
                         onSubmit={undefined}
+                        onValueChange={v => {
+                          if (onValueChange) {
+                            onValueChange(field, v);
+                          }
+                        }}
```

Comment:
suggestion: we could make it a bit more readable with something like this

```
const getValueChangeHandler = (field) => v => onValueChange ? onValueChange(field, v) : undefined;
```

Reply:
IMO it's much less readable :D